### DISCUSSION TITLE
Grupo para el hackathon.-

### DISCUSSION THREAD

**Discussion by Sebastian-Longhitano** [Launch-X-Latam/MisionFrontEnd#504] [Team Posts]
Contento realmente por pertenecer a ésta gran comunidad compañeros!!
Contento por los temas abordados por el grupo de Innovación!!
Estar feliz es lo primero.

Lo segundo es comentarles que me encuentro abierto para conformar grupos con ustedes para el hackathon!

Pueden contactarme a través de:  lx14356@innovaccion.mx

Gracias a todos por leerme! =)
Un cordial saludo!

**Comment by CAMILOITT** [comment]
hola mi nombre es Camilo estoy interesado en hacer equipo
correo: lx11002@innovaccion.mx
